README.md

```diff
@@ -1,13 +1,57 @@
 # Golden Sample Angular App
 
-This example project is a reference implementation to showcase a number of best practices to use when building a new Angular SPA that leverages Backbase components and libraries. The project is regularly updated, so come back often to check for new improvements. For the best experience, these VSCode extenstions for Nx and Jest should be installed. Jest runner allows you to easly run isolated test as you are developing. Nx Console allows you to easily find and run all the possible Nx Commands.
+This golden sample provides examples of the code structure, configuration, and best practices to use for developing your  Backbase journey.
 
-- `nrwl.angular-console`
-- `firsttris.vscode-jest-runner`.
 
-## Important
+## Table of Contents
+* [What is the app for](#what-is-the-app-for)
+* [Prerequisites](#prerequisites)
+* [Authentication details](#authentication-details)
+* [Generate an application](#generate-an-application)
+* [Generate a library](#generate-a-library)
+* [Development server](#development-server)
+* [Code scaffolding](#code-scaffolding)
+* [Build](#build)
+* [Tests](#tests)
+* [Understand your workspace](#understand-your-workspace)
+* [Running with docker](#running-with-docker)
+* [Package as a runnable Docker container](#package-as-a-runnable-docker-container)
+* [Further help](#further-help)
```

Comment:
I do not see all of these sections - do they still have to be completed?

Reply:
All the sections are already covered.